modules/wri_person/config/install/field.field.block_content.person_listing.field_more_link.yml

```diff
@@ -0,0 +1,21 @@
+status: true
+dependencies:
+  config:
+    - block_content.type.person_listing
+    - field.storage.block_content.field_more_link
+  module:
+    - link
+id: block_content.person_listing.field_more_link
+field_name: field_more_link
+entity_type: block_content
+bundle: person_listing
+label: 'More link'
+description: 'Below this listing, link to a page for more info.'
+required: false
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings:
+  title: 2
+  link_type: 17
+field_type: link
```

Comment:
Does this field need a field.storage config file? I thought it might be missing one because the field already exists and we're just tweaking it, but I don't see it on live. Then again, the feature seems to be working without it (on the multidev anyway).

Reply:
Ahhhhh. Great! Thank you for the context!